src/main/java/org/javacord/bot/Main.java

```diff
@@ -31,6 +31,14 @@
      * @throws IOException If there is an error when reading the token file.
```

Comment:
should include that an `IOException` can be thrown if the log4j config file cannot be created (or made more generic).